manifests/virt-handler.yaml.in

```diff
@@ -29,6 +30,8 @@ spec:
         volumeMounts:
         - name: libvirt-runtime
           mountPath: /var/run/libvirt
+        - name: sockets
```

Comment:
I'd like to separate the problem a little: We need cloud-init no question. but is the virt-handler creating it? Or could there be another component which is taking a cloud-init datasource and providing it as a volume to a VM? or - would it even make sense of libvirt is capable of generating noclpoud sources based on the data?

But this should probably be discussed in #340.